src/user/user.controller.ts

```diff
@@ -79,12 +102,12 @@ export class UserController {
     description: 'Tries to delete an user if user with given id exists.',
   })
   @ApiNoContentResponse({
-    description: 'Deletes user with matching id',
+    description: 'User successfully deleted',
```

Comment:
Not sure if added yet, but does deleting the user also delete all their associated data, like donations? This might be a GDPR requirement we'd need to comply with.

Reply:
Deletes all associated data with exception of donations which are soft deleted.

Reply:
Can we implement a second, 'permanent delete' endpoint for the purposes of GDPR-compliant deletion?

Reply:
The user is disassociated from the data, isn't that already GDPR compliant?
I can't implement a second endpoint unless you want one that's generic and will remove all donations with nulls in `user` column.

Reply:
That's an interesting concept, since AFAIR GDPR doesn't forbid nor allow soft-deleting data by disassociating them from a user. Considering donation data can no longer be associated to a user once their ID and other user data is deleted from the database, you could technically say it's *anyone's* data.

Let's stick with that - we can try asking the Dean about this next time 👍 No action needed on your side then.